docs/advanced/peering/offloading-in-depth.md

```diff
@@ -0,0 +1,251 @@
+# Offloading in depth
+
+## Overview
+
+This document will go over the process of acquiring resources and making them available as a `Node` in the consumer cluster.
+
+You can add a `VirtualNode` to a consumer cluster in two different ways:
+
+1. By creating a `ResourceSlice` in the tenant namespace in the consumer cluster.
+2. By creating a `VirtualNode` in the consumer cluster.
+
+Note that the `ResourceSlice` method is the **preferred way** to add a `VirtualNode` to a consumer cluster, but it requires the *Authentication module* to be enabled (that is enabled by default).
+In the following steps, when we are using `ResourceSlices`, we will assume that the *Authentication module* is enabled and the authentication between the clusters is established, either with [`liqoctl peer`](/usage/peer) or with the [manual configuration](/advanced/peering/inter-cluster-authentication).
+
+## Create ResourceSlice
+
+This is the preferred way to add a `VirtualNode` to a consumer cluster.
+
+`````{tab-set}
+
+````{tab-item} liqoctl
+```bash
+liqoctl create resourceslice mypool --remote-cluster-id cool-firefly
+```
+````
+
+````{tab-item} YAML
+```yaml
+apiVersion: authentication.liqo.io/v1beta1
+kind: ResourceSlice
+metadata:
+  annotations:
+    liqo.io/create-virtual-node: "true"
+  creationTimestamp: null
+  labels:
+    liqo.io/remote-cluster-id: cool-firefly
+    liqo.io/remoteID: cool-firefly
+    liqo.io/replication: "true"
+  name: mypool
+  namespace: liqo-tenant-cool-firefly
+spec:
+  class: default
+  providerClusterID: cool-firefly
+status: {}
+```
+````
+
+`````
+
+This command will create a `ResourceSlice` named `mypool` in the consumer cluster, and it will be associated with the `cool-firefly` cluster.
+
+If no resource is specified, the provider cluster will fill them with the default values.
+You can specify the resources you want to acquire by adding:
+
+* `--cpu` to specify the amount of CPU.
+* `--memory` to specify the amount of memory.
+* `--pods` to specify the number of pods.
+
+For other custom resources, you can use the `-o yaml` flag for the `liqoctl create resourceslice` command and edit the `ResourceSlice` spec manifest.
+
+```bash
```

Comment:
I would add a note in the snippet telling where the commands should be executed. Like we do [here](https://github.com/liqotech/liqo/blob/docs/docs/advanced/peering/inter-cluster-authentication.md?plain=1#L64)